greedy_algorithms/kruskals_minimum_spanning_tree.cpp

```diff
@@ -26,6 +64,6 @@ int main() {
         INFINITY, 3,        1,        5,        0,        INFINITY,
         INFINITY, INFINITY, INFINITY, 7,        INFINITY, 0};
 
-    findMinimumEdge(INFINITY, graph);
+    greedy_algorithms::findMinimumEdge(INFINITY, graph);
```

Comment:
could you add tests please :heart:

Reply:
Sure, I can try that. Although I think that fits for another PR, this one's just for adding documentation. 🙂

Reply:
Ping.

Reply:
oh i see